### ISSUE TITLE
audit log does not show deleted comment text

### ISSUE THREAD

**Issue by test2a** [Chocobozzz/PeerTube#3504]
**Describe the current behavior**
i am using gitpod to test some features and i found out that if i create a comment on a video, it correctly displays the comment text along with other details. now, if i delete that same comment, comment text is shown as "" in the audit logs.

**Steps to reproduce:**

1. make a comment.
2. delete that comment
3. check audit logs

audit[12/22/2020, 12:14:27 AM]By root ->comments -> create

{
  "user": "root",
  "domain": "comments",
  "action": "create",
  "comment-id": 1,
  "comment-text": "test comment to check if deleted comments still dont show text",
  "comment-threadId": 1,
  "comment-inReplyToCommentId": null,
  "comment-videoId": 1,
  "comment-createdAt": "2020-12-21T18:44:27.069Z",
  "comment-updatedAt": "2020-12-21T18:44:27.072Z",
  "comment-totalReplies": 0,
  "comment-account-id": 2,
  "comment-account-name": "root"
}

audit[12/22/2020, 12:14:36 AM]By root ->comments -> delete

{
  "user": "root",
  "domain": "comments",
  "action": "delete",
  "comment-id": 1,
  "comment-text": "",
  "comment-threadId": 1,
  "comment-inReplyToCommentId": null,
  "comment-videoId": 1,
  "comment-createdAt": "2020-12-21T18:44:27.069Z",
  "comment-updatedAt": "2020-12-21T18:44:36.916Z",
  "comment-totalReplies": 0,
  "comment-account-id": 2,
  "comment-account-name": "root"
}

**Comment by rigelk** [comment]
This would ideally require soft-deleting comments in the future.

**Comment by test2a** [comment]
yes. this would also solve reversing earlier deletions by admins. if for any reason a comment was deleted, it should be reversible. right now its not

**Comment by Chocobozzz** [comment]
The comment deletion should not be reversible, we can't save information a user deleted.

**Comment by test2a** [comment]
How would an audit log work otherwise? A person wrote something incriminating, it went public,caused damage and before someone could report, the message was deleted. Records should be kept for audit traceability,  to confirm what someone wrote, then edited it, then reedited it and then deleted it.

If not reversible but a copy of all edits, deletions, etc should be kept in the log.

**Comment by Chocobozzz** [comment]
I was just talking about the reversible and soft delete thing. Of course the audit log should keep track of the edits/deletions

**Comment by test2a** [comment]
oh.. i thought you dont want to keep the deleted stuff because that could lead to reversibility and stuff. okay

**Comment by test2a** [comment]
this isnt the place for it but still, @Chocobozzz what are your thoughts on this https://github.com/GRVYDEV/Project-Lightspeed can we use it to improve livestream ingestion speed?

**Comment by rigelk** [comment]
@test2a no we can't, since it's tech stack is too different
